mod.cpp

```diff
@@ -1,12 +1,12 @@
 name = "ACE3";
 picture = "logo_ace3_ca.paa";
-actionName = "Github";
-action = "https://github.com/KoffeinFlummi/ACE3";
-description = "ACE3 - Version 3.0.0";
+actionName = "GitHub";
+action = "https://github.com/acemod/ACE3";
+description = "ACE3 – Version 3.0.0";
```

Comment:
Thanks @Tourorist. 

I'm not sure if Arma supports the hyphen in config files. Unless you have tested this already, I'd suggest you revert that change just to be sure I won't break anything.